### PR TITLE
Link to librt on non-apple platforms

### PR DESCRIPTION
**Pull request by alexcrichton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: rust
 sudo: required
+dist: trusty
```
Comment (sfackler) [inline .travis.yml]: Is this an unrelated change?

Comment (alexcrichton) [inline .travis.yml]: Nah apparently this was needed to get CI to pass, I guess Travis updated some GCE images to 12.04 instead of 14.04?  Not entirely sure why some of the builders went back in time.

**Comment by nagisa** [comment]
This breaks compilation on android. We should really be using build.rs here too.
